### PR TITLE
updated-documentation

### PR DESCRIPTION
**Pull request by arcoprova**
updated documentation on loop page

### CODE REVIEW

File: loop.ftd [hunk 1]
```diff
@@ -13,7 +13,6 @@ Consider this list:
 lang: ftd
 
 \-- list people:
```
Comment (Arpita-Jaiswal) [inline loop.ftd]:
```suggestion
\-- string list people:
```

File: loop.ftd [hunk 2]
```diff
@@ -23,14 +22,14 @@ type: string
 
 -- ft.markdown:
 
-Here we defined a list of type `string`, with three elements. Now say we want to
+Here we defined a list with three elements. Now say we want to
```
Comment (Arpita-Jaiswal) [inline loop.ftd]:
```suggestion
Here we defined a list of type `string`, with three elements. Now say we want to
```

File: loop.ftd [hunk 3]
```diff
@@ -23,14 +22,14 @@ type: string
...
 display each of them:
 
 -- ft.code:
 lang: ftd
 
-\-- ftd.text: ref obj
-$loop$: people as obj
+\-- ftd.text:
```
Comment (Arpita-Jaiswal) [inline loop.ftd]:
```suggestion
\-- ftd.text: $obj
```

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: loop.ftd (hunk 1 above)
Comment (arcoprova) [reply loop.ftd]: done

File: loop.ftd (hunk 2 above)
Comment (arcoprova) [reply loop.ftd]: done

File: loop.ftd (hunk 3 above)
Comment (arcoprova) [reply loop.ftd]: done

**Review by Arpita-Jaiswal — APPROVED** [review]

**Pull request merged**